backend/sozisel/lib/sozisel_web/schema/mutations/presenter_mutations.ex

```diff
@@ -0,0 +1,18 @@
+defmodule SoziselWeb.Schema.Mutations.PresenterMutations do
+  use SoziselWeb.Schema.Notation
+
+  alias SoziselWeb.Schema.Resolvers.PresenterResolvers
+  alias SoziselWeb.Schema.{Middleware}
+
+  object :presenter_mutations do
+    field :send_event_to_participants, :launched_event do
```

Comment:
As mentioned on DM's, I meant the mutation name to be `launch_event` not the resulting type.